src/promptflow/promptflow/contracts/tool.py

```diff
@@ -69,7 +87,15 @@ def from_type(t: type):
             return ValueType.PROMPT_TEMPLATE
         return ValueType.OBJECT
 
-    def parse(self, v):  # noqa: C901
+    def parse(self, v: Any) -> Any:  # noqa: C901
+        """Parse value to the given `ValueType`.
+
```

Comment:
`~promptflow.xxx.ValueType`, and for other places.

Reply:
I found this style `~promptflow.xxx.ValueType` only works in type field, so I do not make changes here.

Reply:
You can use below format to make it work in descritpion field:
![image](https://github.com/microsoft/promptflow/assets/2572521/9cc7b909-0c99-442c-a932-50073159d9b5)

Reply:
Updated.